packages/core/src/storage/index.ts

```diff
@@ -9,10 +9,6 @@ export { SimpleIndexStore } from "./indexStore/SimpleIndexStore.js";
 export * from "./indexStore/types.js";
 export { SimpleKVStore } from "./kvStore/SimpleKVStore.js";
 export * from "./kvStore/types.js";
-export { AstraDBVectorStore } from "./vectorStore/AstraDBVectorStore.js";
-export { ChromaVectorStore } from "./vectorStore/ChromaVectorStore.js";
-export { MongoDBAtlasVectorSearch } from "./vectorStore/MongoDBAtlasVectorStore.js";
-export { PGVectorStore } from "./vectorStore/PGVectorStore.js";
 export { PineconeVectorStore } from "./vectorStore/PineconeVectorStore.js";
```

Comment:
@himself65 
Suggestion: 
Remove all the vector stores here (except SimpleVectorStore) and the user has to import the vector store that he needs
If we do this what are the problems? e.g. how do we make it run with a CJS project?

Reply:
same in cjs module `require(llamaindex/path/to/vectorStore)`